packages/environment-check/src/archive/index.ts

```diff
@@ -6,16 +6,25 @@ import ignore from 'ignore';
 import { t } from '../i18n';
 import { byteNumberToSizeString } from '../formatter';
 
+interface ArchiveProjectOptions {
+    projectRoot: string;
+    targetPath?: string;
```

Comment:
shouldn't it be the folder or dir?
```suggestion
    targetFolder?: string;
```